src/tx/applyTx/checkEpochLength.test.js

```diff
@@ -20,11 +20,14 @@ describe('checkEpochLength', () => {
     const state = getInitialState();
     const epochLength = Tx.epochLength(4);
 
-    checkEpochLength(state, epochLength, {
+    const bridgeState = {
       epochLengths: [4],
-    });
+    };
+
+    checkEpochLength(state, epochLength, bridgeState);
```

Comment:
What will happen if we set epochLength on Tx to number that is not present in ``epochLengths`` array?

Reply:
`checkEpochLength` will fail. It requires the top element of the array to be the same as the one from tx.